playwright-e2e/playwright.config.ts

```diff
@@ -16,8 +16,8 @@ const testConfig: PlaywrightTestConfig = {
   globalSetup: require.resolve('./fixtures/global-setup'),
   testDir: '.',
   testMatch: '**/*.spec.ts',
-  /* Maximum time one test can run for. Test should be short and takes less than 2 minutes to run */
-  timeout: 120 * 1000,
+  /* Maximum timeout per test. Test should be short and takes less than 2 min or 3 min in docker container to run */
```

Comment:
Incr. timeout when tests are running in docker container to avoid timeout failures because sometimes playback speed is slower in docker container.

<img width="987" alt="Screenshot 2023-01-31 at 11 29 53 AM" src="https://user-images.githubusercontent.com/35533885/215826630-64cec613-0dbb-4e7d-9ddd-1eea477b1dcb.png">